examples/funloc/analysis_fun.py

```diff
@@ -118,7 +115,12 @@
     [0, 1],  # Only ensure the standard event counts match
 ]
 
-params.report_params['coil_snr'] = False
+params.report_params.update(  # add a couple of nice diagnostic plots
+    whitening=dict(analysis='All', name='All',
+                   cov='%s-55-sss-cov.fif'),
+    source=dict(analysis='All', name='All',
+                inv='%s-55-sss-meg-eeg-free-inv.fif', times=[0.1, 0.2]),
+)
```

Comment:
If we are just recommending checking onset responses as part of QC then it might be useful to allow `kwargs` for time point and views for the source image?

Reply:
see updated code

Reply:
(and time points are already an option...?)